src/main/java/no/nav/familie/inntektsmelding/typer/dto/ForespørselResultat.java

```diff
@@ -2,5 +2,5 @@
 
 public enum ForespørselResultat {
     FORESPØRSEL_OPPRETTET,
-    IKKE_OPPRETTET_FINNES_ALLEREDE_ÅPEN
```

Comment:
Denne går til fpsak, skal du prodsette de to helt likt? Ellers trenger du vel expand contract her